build failed. looks like it either needs changing in the report or if you want to hack around it you can use `@softer_assert`
```
root: INFO: Location 022027440dc8cbccf3ffa4c334929cba Not Found.
root: ERROR: Error in emitter CouvertureFluff > registered > total_for_region: 'e83f70ec-24e6-4bf6-8dc1-5450e879bdaf'
Traceback (most recent call last):
  File "corehq/ex-submodules/fluff/calculators.py", line 82, in calculate
    if passes_filter else []
  File "corehq/ex-submodules/fluff/emitters.py", line 20, in wrapped_f
    for v in generator:
  File "/mnt/commcare-hq-ro/custom/intrahealth/report_calcs.py", line 75, in total_for_region
    loc = get_location_by_type(form=form, type=u'r\xe9gion')
  File "/mnt/commcare-hq-ro/custom/intrahealth/utils.py", line 135, in get_location_by_type
    if unicode(loc.location_type).lower().replace(" ", "") == type:
  File "/mnt/commcare-hq-ro/corehq/apps/locations/models.py", line 719, in location_type
    "You should use either location_type_name or location_type_object")
  File "/mnt/commcare-hq-ro/corehq/apps/locations/models.py", line 33, in notify_of_deprecation
    _assert(False, message)
  File "/mnt/commcare-hq-ro/corehq/util/soft_assert/core.py", line 33, in __call__
    return self._call(assertion, msg, obj)
  File "/mnt/commcare-hq-ro/corehq/util/soft_assert/core.py", line 47, in _call
    raise AssertionError(msg)
AssertionError: Deprecated Locations feature used: You should use either location_type_name or location_type_object
--------------------- >> end captured logging << ---------------------
```

--- a/custom/intrahealth/utils.py
+++ b/custom/intrahealth/utils.py
@@ -132,7 +132,7 @@ def get_location_by_type(form, type):
             return loc
     for loc_id in loc.lineage:
         loc = Location.get(loc_id)
-        if unicode(loc.location_type).lower().replace(" ", "") == type:
+        if unicode(loc.location_type_name).lower().replace(" ", "") == type:
             return loc
 
 

